### DISCUSSION TITLE
Can't install homebrew on mac

### DISCUSSION THREAD

**Discussion by charlietlamb** [Homebrew/discussions#1684] [Getting started] [answered]
Hey, I've been trying on and off for the last couple of months to install homebrew on my mac and it is not working. I've tried deleting directories, installing and reinstalling homebrew and then not being able to install certain libraries like mongoosedb, I even reset my computer and the problem is still there.
This is the error message i recieve now when trying to reinstall homebrew.

==> Tapping homebrew/core
remote: Enumerating objects: 973663, done.
remote: Counting objects: 100% (340/340), done.
remote: Compressing objects: 100% (189/189), done.
error: 4292 bytes of body are still expected.67 MiB | 6.00 MiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master

Any suggestions as too how I could fix this?
Thanks.

**Comment by carlocab** [comment, marked as answer]
Try checking #666 for a related discussion/solutions.

**Reply by charlietlamb** [reply]
Fixed now. Virgin media child safe was the issue. Thanks for the help
